tests/test_apidoc.py

```diff
@@ -118,6 +120,15 @@ def test_pep_0420_disabled_top_level_verify(make_app, apidoc):
     print(app._status.getvalue())
     print(app._warning.getvalue())
 
+@pytest.mark.apidoc(coderoot=(rootdir / 'root' / 'trailing_underscore'))
```

Comment:
Could you move this to under the `tests/roots/` directory (cf. `tests/roots/test-apidoc`)?
`tests/root/` is commonly used test document. so adding new file affects to time of them.
Sorry for less explanation.